### DISCUSSION TITLE
[Windows] Unexpected Error: SerialException

### DISCUSSION THREAD

**Discussion by sztangwang** [Jakeler/ble-serial#56] [Q&A] [answered]
1、How to use ：ble-serial -d macaddress -m MTU ？
2、tips what caused it？：Unexpected Error: SerialException 

![image](https://user-images.githubusercontent.com/9073767/164022428-b9743aa0-56a4-4730-babe-8eb28e0db3d0.png)

![image](https://user-images.githubusercontent.com/9073767/164022685-88c71269-86f4-452e-be1d-b8099cb25bb8.png)

**Comment by Jakeler** [comment, marked as answer]
1. Your use of -m is correct (if you adapter+device really supports 333 bytes)
2. Looks like you serial port emulator setup is not correct. Since you are on Windows, did you follow https://github.com/Jakeler/ble-serial#additional-steps-for-windows? 
So is com0com properly installed and the port pair created? If yes please also post the com setup script output and from `Program Files (x86)/com0com` dir open `setupc.exe` and run `list`.

**Reply by sztangwang** [reply]
![image](https://user-images.githubusercontent.com/9073767/164132539-c28f63d9-21ab-4a03-aa12-fb685d114523.png)

**Reply by Jakeler** [reply]
Clearly the BLE port pair is missing

**Reply by sztangwang** [reply]
How should I operate？

**Reply by Jakeler** [reply]
just run `ble-com-setup.exe`

**Reply by Jakeler** [reply]
I assume that solved it? if so please "mark as answer" above

**Comment by sztangwang** [comment]
Is this correct？

![image](https://user-images.githubusercontent.com/9073767/164132216-06b11800-0d76-412b-83db-750df659af3f.png)

**Reply by Jakeler** [reply]
No, this just prints the help text, you have to run it without `-h` ;)

**Comment by zeezooX** [comment]
I also have the same error.
![image](https://user-images.githubusercontent.com/26670832/176996276-e96fb0d5-045b-4b13-8b00-6caccee1efc3.png)
However I have setup everything as instructed.
![image](https://user-images.githubusercontent.com/26670832/176996323-8efcb602-d843-407a-a1ee-4cc84a1304f9.png)

**Reply by Jakeler** [reply]
There is also (again) issue #61 open with the same error. Please check and try especially what I wrote in the last 2 comments. If that does not solve it then open another issue, fill out the bug template and put in all results from what you have tried.

In general I recommend Linux, because the setup is much easier and more reliable.
